public/components/datasources/components/manage/accelerations/acceleration_details_flyout.tsx

```diff
@@ -122,7 +132,12 @@ export const AccelerationDetailsFlyout = (props: AccelerationDetailsFlyoutProps)
           </EuiFlexItem>
         </EuiFlexGroup>
         <EuiSpacer size="m" />
-        <EuiTabs style={{ marginBottom: '-25px' }}>{renderTabs()}</EuiTabs>
+        <EuiTabs
+          style={{ marginBottom: '-25px' }}
```

Comment:
I think we should avoid this inline style, and we can switch to a SCSS to define this. But yeah, this is not related to this PR, and we can do that as a follow up.